pkg/services/k8s/apiserver/service.go

```diff
@@ -70,6 +75,7 @@ func (s *service) start(ctx context.Context) error {
 	}
 
 	serverRunOptions := options.NewServerRunOptions()
+	serverRunOptions.Logs.Verbosity = 5
```

Comment:
you might need to change the log adapter support this verbosity level

Reply:
I see. I didn't mean to commit this in. Will revert.